src/Microsoft.AspNetCore.Antiforgery/Internal/IAntiforgeryFeature.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.AspNetCore.Antiforgery.Internal
+{
+    public interface IAntiforgeryFeature
+    {
+        AntiforgeryToken CookieToken { get; set; }
+        bool HaveDeserializedCookieToken { get; set; }
+        bool HaveDeserializedRequestToken { get; set; }
+        bool HaveGeneratedNewCookieToken { get; set; }
+        bool HaveStoredNewCookieToken { get; set; }
+        AntiforgeryToken NewCookieToken { get; set; }
+        string NewCookieTokenString { get; set; }
+        AntiforgeryToken NewRequestToken { get; set; }
+        string NewRequestTokenString { get; set; }
+        AntiforgeryToken RequestToken { get; set; }
```

Comment:
IMO I think this is fine for now that this feature is just a property holder since it's `.Internal`. If we develop more scenarios for this we'll want to do more design work here.